lib/list_export.rb

```diff
@@ -21,6 +21,8 @@ def number_of_rows
       @team.filtered_fact_checks(@parsed_query).count
     when :explainer
       @team.filtered_explainers(@parsed_query).count
+    when :dashboard
+      1
```

Comment:
Please add a comment here to explain why it's hard-coded as "1".

Reply:
Done